### DISCUSSION TITLE
Number users in a room (how to show it on a landing page)

### DISCUSSION THREAD

**Discussion by yengalvez** [mozilla/hubs#4998] [Hubs Cloud Questions] [answered]
Hi guys! I need your help
It's very simple. I only need a way to know how many people are inside a room to show it in the landing page
I need it to show what room is empty, something like 5/25 -> Join
Do you know how I can do it?
I think it is a base functionality, but I don't know how to implement or access to this information
thanks!!

**Comment by pattersonbl2** [comment, marked as answer]
We currently don't have a feature that will do this.  This could be something designed with a custom setup. It would require a bit of work and research into the open-source code.
